EmuELEC/EmuELEC
# dev - duckstation - auto gamepad - analog fix

Fixes for duckstation with Analog Controllers.

To test SSH in and run the following commands:
```
cd /emuelec/bin
wget https://raw.githubusercontent.com/EmuELEC/EmuELEC/1957cc9163b4ec3b7c19be937573c6446dca9602/packages/sx05re/emulators/duckstation/scripts/set_duckstation_joy.sh
chmod +x /emuelec/bin/set_duckstation_joy.sh
```
Make sure duckstation auto gamepad is enabled in ES Emuelec Settings and it should now use the test version.
